spec/models/user_spec.rb

```diff
@@ -1,5 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validations' do
+    it { should validate_presence_of :email }
```

Comment:
I like these being on one line, but the word should instead of expect anywhere is offputting to me. Don't know why.